src/redismodule.h

```diff
@@ -63,7 +63,7 @@
 #define REDISMODULE_HASH_DELETE ((RedisModuleString*)(long)1)
 
 /* Error messages. */
-#define REDISMODULE_ERRORMSG_WRONGTYPE "WRONGTYPE Operation against a key holding the wrong kind of value"
+#define REDISMODULE_ERRORMSG_WRONGTYPE "WRONGTYPE Operation against a key holding the wrong kind of value or key already exists"
```

Comment:
We should not change this but rather separate the cases and return the appropriate message for each one of them. Please remove.